src/anshuKJun20/ExtractItemIdFromString.java

```diff
@@ -0,0 +1,61 @@
+package anshuKJun20;
+
+public class ExtractItemIdFromString {
+
+	void findItemId(String input) {
+	//first Apprach;
+	   /* String replaceStr = input.replace("/", "&");
+		String[] array = replaceStr.split("&");
+		int num = 0;
+		for (int index = 0; index < array.length; index++) {
+			try {
+				if (array[index].contains("itemid=12345")) {
+					String[] temparray = array[index].split("=");
+					for (int innerindex = 0; innerindex < temparray.length; innerindex++) {
+						try {
+							num += Integer.parseInt(temparray[innerindex]);
+						} catch (NumberFormatException ne) {
+						}
+					}
+				}
+			} catch (Exception e) {
+			}
+		}
+		System.out.println(" Item id is " + num);**/
+		
+		// second Approach
+		String sum = ""; int count = 0;
+		if(input.contains("itemid=")) {
+		int index = input.indexOf("12345");
```

Comment:
You can find substring like lastIndexOf("itemId=").Now try to find item id value without using character class method.
Either you can use replace ("itemId=", "")  or split method.

Reply:
Changes has been done.